lib/components/narrative/line-itin/itin-summary.js

```diff
@@ -82,6 +82,19 @@ export default class ItinerarySummary extends Component {
     itinerary: PropTypes.object
   }
 
+  getIcon = (iconId, customIcons) => {
+    // Check if there is a custom icon
+    if (customIcons && iconId in customIcons) {
```

Comment:
Is there a way we can get rid of this `customIcons` variable altogether? It would be nice to have everything inside the mode icon component.

Reply:
Working on it. I will shortly push a PR against OTP-ui to address that.

Reply:
This is resolved with opentripplanner/otp-ui#119.